docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -194,15 +194,15 @@ As parameters for **AbstractNoncedBox** three template parameters shall be provi
 ::
  P extends Proposition
 
-- Proposition type for the box, for common purposes. ``PublicKey25519Proposition`` could be used as it used in regular boxes
+- Proposition type for the box, for common purposes. ``PublicKey25519Proposition`` could be used as it is used in regular boxes
   ::
    BD extends AbstractNoncedBoxData<P, B, BD>
    
 - Definition of type for Box Data which contains all custom data for a new custom box
   ::
    B extends AbstractNoncedBox<P, BD, B>
 
-- Definition of type for Box itself, required for description inside of new Custom Box data.
+- Definition of type for the Box itself, required for description inside of new Custom Box data.
```

Comment:
@zennermac - The definition of the type for the box itself is required for the description inside of the new Custom Box Data.